src/components/layout/yearInReview/2021/intro.tsx

```diff
@@ -20,49 +23,92 @@ const Intro: React.FC = () => {
       />
       <div className="px-5 mx-auto pb-10">
         <SubSection
-          header="We explored and grew so much this year"
+          header={intl.formatMessage({
+            id: 'page.year-in-review.2021.section.we-grew.intro.heading',
+            defaultMessage: 'We explored and grew so much this year',
+          })}
           headerSize="3xl"
           contentSize="2xl"
         >
-          This year, we refined our goals and built projects that focused
-          primarily on filling existing gaps in our movement. We grew more
-          meaningful partnerships and we greatly improved our branding, design
-          team, and services offered. We&apos;re excited to show you what
-          we&apos;ve done below!
+          <FormattedMessage
+            id="page.year-in-review.2021.section.we-grew.intro.paragraph"
+            defaultMessage="This year, we refined our goals and built projects that focused primarily on filling existing gaps in our movement. We grew more meaningful partnerships and we greatly improved our branding, design team, and services offered. We're excited to show you what we've done below!"
+          />
         </SubSection>
       </div>
 
       <div>
         <HighlightBlock
           borderColor="magenta"
-          headerStart="We launched"
-          headerBold="8 new projects"
-          headerEnd="for the movement"
+          headerStart={intl.formatMessage({
+            id: 'page.year-in-review.2021.section.we-grew.projects.heading.0',
+            defaultMessage: 'We launched',
+          })}
+          headerBold={intl.formatMessage({
+            id: 'page.year-in-review.2021.section.we-grew.projects.heading.1',
+            defaultMessage: '8 new projects',
+          })}
+          headerEnd={intl.formatMessage({
+            id: 'page.year-in-review.2021.section.we-grew.projects.heading.2',
+            defaultMessage: 'for the movement',
+          })}
```

Comment:
Should I remove the translations for `headerStart`, `headerBold`, and `headerEnd` too @MatsThieme?

Reply:
No problem, deleted them in my last commit